### ISSUE TITLE
Can't export images

### ISSUE THREAD

**Issue by sguldbra2** [imperial-photonics/FLIMfit#222]
[fit nanorod_1to100_exc628_em692_pexc15p8uw_pdep4mw_t30_1 tau_1 merge.pdf](https://github.com/imperial-photonics/FLIMfit/files/247377/fit.nanorod_1to100_exc628_em692_pexc15p8uw_pdep4mw_t30_1.tau_1.merge.pdf)

When I try to export images, they look strange. Please see attachment. Tif-files look thesame.

**Comment by imunro** [comment]
I attempted to reproduce this with Stina's data on my Mac and I see the oversized colour bar ( I suspect this a a Matlab default as the images are small 64x64) but don't get any text on the resulting images.
I repeated it with larger images and get a normal looking colour-bar but still no text.

Hard to say which behaviour is better- after all adding your own text is annoying but easier than removing oversized text.
